build.gradle

```diff
@@ -17,9 +11,17 @@ buildscript {
     classpath "com.github.jengelman.gradle.plugins:shadow:2.0.4"
     classpath "com.google.protobuf:protobuf-gradle-plugin:0.8.1"
     classpath "gradle.plugin.nl.javadude.gradle.plugins:license-gradle-plugin:0.14.0"
+    classpath "org.shipkit:shipkit:2.0.13"
```

Comment:
what doe this do?

Reply:
It's the dependency which handles pushing to bintray which we use to sync to maven central